File: WebApplication1/Domain/Models/Report.cs
namespace Domain.Models;

public class Report
{
    public int Id { get; set; }
    public int UserId { get; set; }
    public string Description { get; set; }
    public int ReviewId { get; set; }

    public Report(int userId, string description, int reviewId)
    {
        UserId = userId;
        Description = description;
        ReviewId = reviewId;
    }
}

File: Java/JavaDatabaseServer/src/main/java/com/model/Report.java
package com.model;

public class Report {
    private int id;
    private int userId;
    private String description;
    private int reviewId;

    public Report(int id, int userId, String description, int reviewId) {
        this.id = id;
        this.userId = userId;
        this.description = description;
        this.reviewId = reviewId;
    }

    public int getId() {
        return id;
    }

    public int getUserId() {
        return userId;
    }

    public String getDescription() {
        return description;
    }

    public int getReviewId() {
        return reviewId;
    }

    @Override
    public String toString() {
        return "Report{" +
                "id=" + id +
                ", userId=" + userId +
                ", description='" + description + '\'' +
                ", reviewId=" + reviewId +
                '}';
    }
}
